Community standard documents (#41)
kolibrios-nextgen/kolibrios-nextgen: README.md

# KolibriOS-NG

KolibriOS-NG is a fork of the [KolibriOS](https://kolibrios.org/) operating system

## Overview

KolibriOS is a hobby operating system for x86-compatible computers, its kernel is written entirely in [Fasm](https://flatassembler.net/) assembly language. Based on [MenuetOS](https://www.menuetos.net/), it uses its own standards and is not fully POSIX or UNIX compliant

## Contributing

> A detailed description of the contribution process for those interested is described in the [contributing guide](./CONTRIBUTING.md)

Reviewer: Might be a good idea to rephrase to avoid repetition, for instance:
```suggestion
Detailed instructions for the contribution process can be found in the [contributing guide](./CONTRIBUTING.md)
```

> KolibriOS is a hobby operating system for x86-compatible computers, its kernel is written entirely in [Fasm](https://flatassembler.net/) assembly language. Based on [MenuetOS](https://www.menuetos.net/), it uses its own standards and is not fully POSIX or UNIX compliant.

Reviewer 2: No link to documentation

Author: Fixed

## Special thanks

The KolibriOS-NG team expresses special thanks to the author of the 32-bit __MenuetOS__, [Ville Turjanmaa](https://www.menuetos.net/contact.htm). We also want to note that all __MenuetOS__ copyrights have been preserved.

## License

> Contents of this repository licensed under terms of __GNU GPL 2.0__ unless otherwise specified. See [this](COPYING.TXT) file for details

Reviewer: ```suggestion
Contents of this repository are licensed under the terms of __GNU GPL 2.0__ unless otherwise specified. See [this](COPYING.TXT) file for details
```
